pact/pact.py

```diff
@@ -214,6 +214,8 @@ def start_service(self):
             command.extend(['--sslcert', self.sslcert])
         if self.sslkey:
             command.extend(['--sslkey', self.sslkey])
+        if self.cors:
+            command.extend(['--cors=*'])
```

Comment:
Why the `=*`? I think just passing this without arguments (i.e. treat it as a flag) is all that is needed.

Or is this something specific to the way the python CLI lib interacts?

Reply:
I tested it without passing `=*` and it works fine.

The documentation of the mock server tells me that an argument must be provided

`pact\bin\pact\bin\pact-mock-service.bat help start`
```
 -o, [--cors=CORS]  # Support browser security in tests by responding to OPTIONS requests and adding CORS headers to mocked responses
```

But because `self.cors` is a boolean there is no way to pass different options for the cors parameter and it is
therefore better to append `--cors` without any value.